src/components/Button/styles.ts

```diff
@@ -0,0 +1,38 @@
+import { css } from '@emotion/css';
+
+const small = '0.25rem 0.5rem';
+const medium = '0.5rem 1rem';
+const large = '1.5rem 2rem';
```

Comment:
```suggestion
const SMALL_WHAT = '0.25rem 0.5rem';
const MEDIUM_WHAT = '0.5rem 1rem';
const LARGE_WHAT = '1.5rem 2rem';
```
be clear in naming some variables, and make sure there is a different between a constant and something processed. This variable will never ever changed in the runtime right? use `SCREAMING_SNAKE_CASE` in that case (or you guys decide).